cmd/api/src/model/auth.go

```diff
@@ -601,7 +562,7 @@ type UserSession struct {
 	User             User `gorm:"constraint:OnDelete:CASCADE;"`
 	UserID           uuid.UUID
 	AuthProviderType SessionAuthProvider
-	AuthProviderID   int32
+	AuthProviderID   int32 // This is the SSO Provider ID if SSO session
```

Comment:
Typo maybe? Did you mean : `// This is the SSO Provider ID of SSO session`

Reply:
Opted to rephrase it completely 👍